ramlfications/utils.py

```diff
@@ -54,24 +56,39 @@ def load_schema(data):
     return data
 
 
-def requests_download_xml():
+def requests_download(url):
```

Comment:
but not sure if that's even the best way to go about it - to force `urllib_download` for files - at least within `utils.py` logic.